mechanics/RELEASE-SCHEDULE.md

```diff
@@ -0,0 +1,16 @@
+# Knative Release Schedule
+
+Knative releases every 6 weeks. As much as possible, releases should be driven by automation, and repos should be ready to release at any point. It should also be possible to produce and consume nightly artifacts.
+
+With that said, it can be useful to have a list of when future releases will happen, so this document provides a schedule for the next 6+ months of releases.
+
+| Release | Date |
+| ------- | ---- |
+| 0.17    | 2020-08-18 |
+| 0.18    | 2020-09-29 |
+| 0.19    | 2020-11-10 |
+| 0.20    | 2020-01-05 ** Moved by 2 weeks for end of year holidays** |
+| 0.21    | 2020-02-02 |
```

Comment:
I thought we agreed on staggering those, no? so 02/16 and so forth?

Reply:
It's 6 weeks from the original release date of 2020-12-22

Reply:
Exactly, when we talked about this last it would've been 6 weeks from the delayed release.

Reply:
+1, 4 weeks may be too small a window, let's shift everything by 2 weeks.

(One reason in the meeting we didn't want to move to 12-08 was to avoid having 4 weeks for v01.9)

Reply:
Updated.